docs/elements.md

```diff
@@ -946,6 +946,7 @@ Provides an in-browser rich text editor, aimed mostly at manual grading essay-ty
 | `placeholder`        | string  | "Your answer here" | Text to be shown in the editor as a placeholder when there is no student input.                                                                                                                                                                                                                                                                                                                                                                       |
 | `format`             | string  | `html`             | Format used to save the student's response. The element supports `html` and `markdown` formats. This format also affects how the source file name or inner HTML is interpreted.                                                                                                                                                                                                                                                                       |
 | `markdown-shortcuts` | boolean | `true`             | Whether or not the editor accepts shortcuts based on markdown format (e.g., typing `_word_` causes the word to become italic).                                                                                                                                                                                                                                                                                                                        |
+| `counter`            | string  | `none`             | Display a `word` or `character` count. This measure is inaccurate for unicode characters.                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
```suggestion
| `counter`            | `word`, `character` or `none`  | `none`             | Display a `word` or `character` count. This measure is inaccurate for unicode characters.                                                                                                                                                                                                                                                                                                                                                             |
```

This will require running `make format` to reformat the table. Again, bonus points if you do the same for `format`.